tensorflow/tsl/profiler/lib/BUILD

```diff
@@ -259,6 +259,7 @@ cc_library(
     deps = [
         "@com_google_absl//absl/strings",
         "//tensorflow/tsl/platform:types",
+	"//tensorflow/compiler/xla/backends/profiler/gpu:nvtx_utils",
```

Comment:
I moved the code to tensorflow/compiler/xla/backends/profiler/gpu/nvtx_utils.* as discussed offline.
But this create this dependency that fell in the wrong direction.
To my knowledge, XLA depends on TSL. But the reverse shouldn't happen.

So I'll see to move the new feature in nvtx_utils.* to tensorflow/tsl/profiler/lib/nvtx_utils.*
If you prefer something else, just tell me.